src/js/components/AppConfigEditFormComponent.jsx

```diff
@@ -105,6 +106,13 @@ var AppConfigEditFormComponent = React.createClass({
     });
   },
 
+  setViewVolumes: function (isVolumesOpen = true) {
+    isVolumesOpen = isVolumesOpen === true ? true : false;
```

Comment:
Am I missing something or you could simply `this.setState({isVolumesOpen: !!isVolumesOpen})` ?